### PR TITLE
Moves cross_address/allowed to the config. Merges the protected config into the regular.

### PR DESCRIPTION
**Pull request by Cyberboss**
The reason it had for not being there seems to not be the case anymore

### CODE REVIEW

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
You must set the configs vv/sdql procs to deny changes to these values

They are protected to keep people with VV from being able to leak the coms key by setting the value to a server they control.

**Comment by Cyberboss** [comment]
I have 0 SDQL knowledge how do you prevent that?

**Comment by MrStonedOne** [comment]
there is already a version of the sdql proc for the protected config (speaking of that, get rid of that, move all of that to the main config)

**Comment by optimumtact** [comment]
SDQL and VIEWVAR use the same check proc don't they? I'm sure @kevinz000 set that up

**Comment by MrStonedOne** [comment]
No, they should, but they don't

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -901,3 +903,8 @@
 		statclick = new/obj/effect/statclick/debug("Edit", src)
 
 	stat("[name]:", statclick)
+
+/datum/configuration/vv_edit_var(var_name, var_value)
+	if(var_name == "cross_address" || var_name == "cross_allowed")
```
Comment (MrStonedOne) [inline code/controllers/configuration.dm]: Make this a switch statement

**Comment by Cyberboss** [comment]
Depends on #24854

**Pull request merged**